source/assets/css/visual-design/_typography.scss

```diff
@@ -179,8 +179,10 @@ code {
   font-size: ($sl-font-size--small / $sl-font-size--medium) * 1em;
   line-height: 1;
 
-  nav &,
-  p & { white-space: nowrap; }
+  nav & { white-space: nowrap; }
+  p & {
+    @include sl-breakpoint--x-large { white-space: nowrap; }
```

Comment:
I think a medium breakpoint might make more sense here. It's really mobile screens which are likely to suffer from the nowrap behavior, right?

Reply:
The issue gets worse in lower resolutions, but in this case overflow can occur in resolutions as high as 1200px. 
![1200](https://user-images.githubusercontent.com/610974/173064231-b847531c-188d-4222-af83-a6bfc4e691f2.png)
For comparison, screenshots at medium (640px) and large (960px) breakpoints:
![640](https://user-images.githubusercontent.com/610974/173064942-dde92f30-4d82-4a05-9209-ea5d4d35103e.png)
![960](https://user-images.githubusercontent.com/610974/173064950-065e3b50-7c11-4aa7-8755-0397ccc7d74c.png)